static/src/js/components/ProjectPanels/AccessMethod.js

```diff
@@ -14,102 +14,155 @@ import './AccessMethod.scss'
-      <ProjectPanelSection heading="Select Data Access Method">
-        <div className="access-method__radio-list">
-          <RadioList defaultValue="Download">
-            <Radio
-              id={`${id}_access-method__direct-download`}
-              name={`${id}_access-method__direct-download`}
-              value="Download"
-              checked
+export const AccessMethod = ({
+  index,
+  metadata,
+  onSelectAccessMethod,
+  onSetActivePanel
+}) => {
+  const {
+    id,
+    tags
+  } = metadata
+
+  const handleAccessMethodSelection = (method) => {
+    console.log('handleAccessMethodSelection', method)
+    onSelectAccessMethod({
+      collectionId: id,
+      selectedAccessMethod: method
+    })
+  }
+
+  const collectionCapibilities = tags['edsc.extra.serverless.collection_capabilities']
+
+  const { data = {} } = collectionCapibilities
+  const { granule_online_access_flag: downloadable } = data
+
+  const radioList = []
+
+  if (downloadable) {
+    radioList.push((
+      <Radio
+        id={`${id}_access-method__direct-download`}
+        name={`${id}_access-method__direct-download`}
+        key={`${id}_access-method__direct-download`}
+        value="Download"
+        onClick={() => handleAccessMethodSelection('download')}
+      >
+        Direct Download
+        <OverlayTrigger
+          placement="right"
+          overlay={(
+            <Tooltip
+              className="tooltip--large tooltip--ta-left"
             >
-              Direct Download
-              <OverlayTrigger
-                placement="right"
-                overlay={(
-                  <Tooltip
-                    className="tooltip--large tooltip--ta-left"
-                  >
-                    Direct download of all data associated with the selected granules.
-                    The desired data will be available for download immediately.
-                    Files will be accessed from a list of links displayed in the
-                    browser or by using a download script.
-                  </Tooltip>
-                )}
-              >
-                <i className="access-method__radio-tooltip fa fa-info-circle" />
-              </OverlayTrigger>
-            </Radio>
-            <Radio
-              id={`${id}_access-method__stage-for-deliver`}
-              name={`${id}_access-method__stage-for-deliver`}
-              value="Stage For Delivery"
+              Direct download of all data associated with the selected granules.
+              The desired data will be available for download immediately.
+              Files will be accessed from a list of links displayed in the
+              browser or by using a download script.
+            </Tooltip>
+          )}
+        >
+          <i className="access-method__radio-tooltip fa fa-info-circle" />
+        </OverlayTrigger>
+      </Radio>
+    ))
+  }
+
+  if (Object.keys(tags).indexOf('edsc.extra.serverless.subset_service.echo_orders') !== -1) {
```

Comment:
Within `util/tags.js` we have `getValueForTag` -- we should add an additional util for `hasTag` that does this for us, it will be used in multiple places. 1.0 implements find buy value as well, I'm not sure we'll need that quite yet

```  
  def self.has_tag(collection, tag, value=nil)
    tag = tag_key(tag)
    tags = collection['tags']
    actual = tag_value(collection, tag)
    if tags.is_a?(Array) || !value.nil?
      (value.nil? && actual != nil) || (!value.nil? && actual == value)
    else
      !tags.nil? && !tags[tag].nil?
    end
  end
```